Safety/Safety-Systems/AuxTel-Safety-Gate-Introduction.rst

```diff
@@ -0,0 +1,199 @@
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+
+################################
+AuxTel Safety Gate Introduction
+################################
+
+
+.. _AuxTel-Safety-Gate-Introduction-Overview:
+
+Overview
+=========
+
+This document provides an introduction to the AuxTel safety gate, 
+including its components, functional states, and the emergency safety mechanisms in place.
+
+**AuxTel safety gate** is an electronic gateway designed to prevent the telescope or dome from moving 
+while people are on the 2nd floor. Is installed at the bottom of the stairs 
+and controlled by the AuxTel Interlock System (AuxTel IS) located in the Main cabinet.
+
+
+.. _AuxTel-Safety-Gate-Introduction-Components:
+
+Components
+============
+
+The **main components** of the gate are:
+
+#. **Frame**, the mechanical frame of the gate attached to the floor.
+#. **Gate** can be opened and closed like a regular gate.
+#. **Handle** used to physically open/close the gate, and to attach the gate to the frame.
+#. **Control buttons** to open/close the gate electronically.
+#. **Status Indicators** LEDs that indicate the gate status.
+
+.. Fig2. Safety gate components.
+
+.. figure:: ./_static/safety-gate-components.png
+      :width: 250px
+      
+      Safety gate components
+
+
+.. _AuxTel-Safety-Gate-Introduction-Gate-Functional-States:
+
+Gate Functional States 
+===========================
+
+.. - section 2
+
+The gate has three functional states: 1: Open, 2: Closed, 3: Error.
+
+.. _AuxTel-Safety-Gate-Introduction-Gate-Functional-States-Open:
+
+Open
+-----------
+
+When the gate is open, its electronics send an **interlock signal** to the AuxTel IS,
+**disabling the drives** for the telescope and dome. 
+
+Consequently, neither the telescope nor the dome can be operated with the gate open. 
+This ensures that people working on the 2nd floor are safe from unexpected movement.
+
+.. _AuxTel-Safety-Gate-Introduction-Gate-Functional-States-Closed:
+
+Closed
+------------
+When the gate is closed, its electronics **clear the interlock** to the AuxTel IS, 
+**enabling the drives** for the telescope and dome drives. 
+
+A closed gate indicates that no one is on the 2nd floor, making it safe to allow movements.
+
+
+.. warning::
+    It is physically possible to go to the 2nd floor without opening the gate 
+    (by entering from the stairs side), but this violates safety protocols 
+    and should **NOT** be done under normal circumstances.
+
+
+.. _AuxTel-Safety-Gate-Introduction-Gate-Functional-States-Error:
+
+Error
+----------
+
+In error mode **the gate cannot be locked again**. 
+This usually occurs after the emergency release has been activated. 
+Next section this mechanism is described.
```

Comment:
I think this would sound better and also keeps the passive voice:
This mechanism is described in the next section.

Reply:
I changed line 106 to  "This mechanism is described in the next section" is there something else I should change?